pkg/client/userd/daemon/grpc.go

```diff
@@ -155,6 +155,14 @@ func (s *Service) Status(c context.Context, ex *empty.Empty) (result *rpc.Connec
 			})
 		} else {
 			result = s.session.Status(s.sessionContext)
+			err = s.WithSession(c, "status", func(ctx context.Context, session userd.Session) error {
+				status, err := session.ManagerClient().Status(c, &empty.Empty{})
+				if err != nil {
```

Comment:
If a new client connects to an older traffic-manager, then the error here will be `codes.Unimplemented`. This must be OK. It must not propagate as an error.